### PR TITLE
Export raw container metrics

### PR DESCRIPTION
**Pull request by vmarmol**
This mostly fixes https://github.com/GoogleCloudPlatform/kubernetes/issues/490

Tested on a live cluster and the data is exported as expected. We get usage of the Kubelet! :D

/cc @vishh

### CODE REVIEW

**Comment by vmarmol** [comment]
Jenkins failures are expected, Jenkins is broken right now.

**Comment by vishh** [comment]
@vmarmol: I am fixing e2e. Kindly bear with me for some time.

**Comment by vmarmol** [comment]
Yeah np :) let me know if I can help

**Comment by vmarmol** [comment]
Build is green after a rebase!

File: sources/datasource/types.go [hunk 1]
```diff
@@ -40,6 +40,9 @@ type Kubelet interface {
 	// GetContainer returns container spec and stats for the container pointed to by 'host.Resource', running on the kubelet specified in 'host.IP'.
 	// TODO(vishh): Once kubelet exposes a get all stats API, modify this API to return stats for all Pods.
 	GetContainer(host Host, start, end time.Time, resolution time.Duration) (containers *api.Container, err error)
+
+	// Get stats for all non-Kubernetes containers.
+	GetAllContainers(host Host, start, end time.Time, resolution time.Duration) ([]api.Container, error)
```
Comment (vishh) [inline sources/datasource/types.go]: Why not combine the two methods into one?

Comment (vishh) [inline sources/datasource/types.go]: nvm. That requires kubelet change.

Comment (vmarmol) [inline sources/datasource/types.go]: Done!

**Comment by vishh** [comment]
LGTM. Ignoring travis since jenkins runs unit tests.

**Pull request merged**